incubator/elasticsearch/values.yaml

```diff
@@ -3,45 +3,52 @@
 # Declare variables to be passed into your templates.
 
 image:
-  repository: "jetstack/elasticsearch-pet"
-  tag: "2.4.0"
+  repository: "centerforopenscience/elasticsearch"
```

Comment:
If you change the default container, please update the README. Thx!

Reply:
done

Reply:
👍

Reply:
Do we need a `LICENSE` for docker files?